### ISSUE TITLE
Outside of react component

### ISSUE THREAD

**Issue by sm2017** [use-global-hook/use-global-hook#54]
@andregardi How can I access store , and dispatching actions outside of react components?

**Comment by huy23121994** [comment]
> @andregardi How can I access store , and dispatching actions outside of react components?

same question, lol

**Comment by gor918** [comment]
> @andregardi How can I access store , and dispatching actions outside of react components?

@andregardi, I think it will be better to have more examples in Readme to avoid similar questions.

**Comment by ketan-10** [comment]
@huy23121994 Do you mean is it already answered? If so could you please share?

**Comment by iRON5** [comment]
Oh, sorry guys, but I think the project is totally dead, since my last request to get access to npm there was no answer yet and there are some PRs that live longer than 2 yrs.

Probably someday I'd get enough access here and will be able to publish new updates on this repo, but for now I'd suggest you just replace this packages with one of [more popular alternatives](https://github.com/use-global-hook/use-global-hook/issues/73#issuecomment-963307115)

**Comment by no-stack-dub-sack** [comment]
@iRON5 yup, considered this package as well. and conceptually it was super easy and fun to use, but just not enough support and a bit too buggy in more complex situations to risk using in any kind of non-pet / production project.

transitioned to using easy-peasy before reading your list of suggested alternatives and while it does not appear to have a ton of support currently, either. it's most recent version, version 5.0, seems to be quite stable.
